nuxeo-services/login/nuxeo-platform-login-token-rest/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/login/tokenauth/AuthenticationTokensObject.java

```diff
@@ -18,29 +18,34 @@
  */
 package org.nuxeo.ecm.restapi.server.jaxrs.login.tokenauth;
 
-import org.nuxeo.ecm.core.api.DocumentModel;
-import org.nuxeo.ecm.core.api.DocumentModelList;
-import org.nuxeo.ecm.core.api.NuxeoPrincipal;
-import org.nuxeo.ecm.tokenauth.io.AuthenticationToken;
-import org.nuxeo.ecm.webengine.model.WebObject;
-import org.nuxeo.ecm.webengine.model.impl.AbstractResource;
-import org.nuxeo.ecm.webengine.model.impl.ResourceTypeImpl;
-import org.nuxeo.ecm.tokenauth.service.TokenAuthenticationService;
-import org.nuxeo.runtime.api.Framework;
+import java.util.Calendar;
```

Comment:
Please avoid reorganizing imports in the same commit as the features/fixes being implemented themselves.

Reply:
Restored order.